aspects/electrical/Converter/GunnsElectConverterInput.cpp

```diff
@@ -452,12 +452,13 @@ GunnsBasicLink::SolutionResult GunnsElectConverterInput::confirmSolutionAcceptab
         mInputVoltageValid = false;
     } else if (0 == convergedStep) {
         /// - We only check for solution rejection and state change after the network has converged.
-        ///   Until it has converged, we assume hte input voltage is valid.
+        ///   Until it has converged, we assume the input voltage is valid.
         mInputVoltageValid = true;
     } else {
         /// - Sensors are optional; if a sensor exists then the trip uses its sensed value of the
         ///   truth parameter, otherwise the trip looks directly at the truth parameter.
-        float sensedVin = MsMath::limitRange(-FLT_MAX, mPotentialVector[0], FLT_MAX);
+        float sensedVin = static_cast<float>(
+            MsMath::limitRange(static_cast<double>(-FLT_MAX), mPotentialVector[0], static_cast<double>(FLT_MAX)));
```

Comment:
i'd rather demote mPotentialVector[0] to float here than promote FLT_MAX, and we don't need to do this comparison as doubles.  But we don't have an overloaded MsMath::limitRange for floats.  I'm not going to make you do that.  I'll accept this now and take an action to change it later.